docs/contribute/before_pr.md

```diff
@@ -0,0 +1,7 @@
+# Checklist before you open a pull request
+   
+ * In the main `nodecg-io` directory run `npm run lint` and fix all errors and warnings that are displayed.
+ 
+ * In the main `nodecg-io` directory run `npm run format`
+ 
+ * Now you can create your pull request
```

Comment:
I don't find this very usefull. 
Linting is done automatically through github actions on pull requests, so no need to check this twice.

Also stuff like this would better be included in a contributing.md or a pull request template.

Still complaining on a high level 😅